### PR TITLE
doc: add minutes for 08 april 2020

### PR DESCRIPTION
**Pull request by gireeshpunathil**

### CODE REVIEW

**Comment by mmarchini** [comment]
Screenshot from 2020-04-10 12-08-57

The formatting is off (I noticed that on previous notes as well).

**Comment by gireeshpunathil** [comment]
@mmarchini - thanks, fixed it!

File: wg-meetings/2020-04-08.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+## Links
+
+* **Recording**: Unavailable
+* **GitHub Issue**: https://github.com/nodejs/diagnostics/issues/374
+* **Minutes Google Doc**: https://docs.google.com/document/d/1hD30I0WsXxOrDDp-2RUk1Bb4T5lsMArwYoAwA_021lc/edit
+
+## Present
+
+* Harshitha K P (@harshithakp)
+* Diagnostics team: @nodejs/diagnostics
```
Comment (mhdawson) [inline wg-meetings/2020-04-08.md:12]: Looks like this is missing some people who were ther?

Comment (gireeshpunathil) [reply wg-meetings/2020-04-08.md]: looks like we missed to register our attendance. I added those now based on what I recollect,  so team - if this is incorrect, feel free to propose edits here (actually I joined a bit late due to overlap with tsc, and left early)

**Comment by gireeshpunathil** [comment]
@mmarchini - addressed your comment, PTAL!

**Comment by gireeshpunathil** [comment]
landed in 5bba7eea16afdbe2497e04725965a795a435c1be

**Pull request closed without merging**